django_enum_choices/utils.py

```diff
@@ -15,3 +24,25 @@ def value_from_built_choice(built_choice):
         return built_choice[0]
 
     return built_choice
+
+
+def build_enum_choices(
+    enum_class: Enum,
+    choice_builder: Callable
+) -> Tuple[Tuple[str]]:
+    choices = [
+        choice_builder(choice)
+        for choice in enum_class
+    ]
+
+    if not all(
+        isinstance(value, str) for value
+        in chain.from_iterable(choices)
+    ):
+        raise EnumChoiceFieldException(
```

Comment:
Lets be more specific & user friendly here. Lets actually say what type we found + the repr of the element :thinking: Might be helpful for debugging